website/www/site/data/en/quotes.yaml

```diff
@@ -10,14 +10,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO:
-# Placeholder texts should be updated later
 - text: A framework that delivers the flexibility and advanced functionality our customers need.
   icon: icons/quote-icon.svg
-  logoUrl: images/quote-paypal-logo.png
+  logoUrl: images/logos/full-color/nameless/eBay.png
 - text: A framework that delivers the flexibility and advanced functionality our customers need.
-  icon: icons/quote-icon.svg 
-  logoUrl: images/quote-paypal-logo.png
+  icon: icons/quote-icon.svg
+  logoUrl: images/logos/full-color/nameless/klio.png
```

Comment:
Shouldn't each quote be changed too? This looks like we're attributing the same quote to three different entities.

Reply:
Yes I added it as TODOs

Reply:
Thanks!